Lib/ipaddress.py

```diff
@@ -975,6 +972,29 @@ def is_multicast(self):
         return (self.network_address.is_multicast and
                 self.broadcast_address.is_multicast)
 
+    @staticmethod
+    def _is_subnet_of(a, b):
+        # Always false if one is v4 and the other is v6.
+        if a._version != b._version:
+            raise TypeError("{} and {} are not of the same version"
+                            .format(a, b))
+        # Dealing with another network.
+        if hasattr(b, 'network_address') and hasattr(b, 'broadcast_address'):
```

Comment:
Since `_is_subnet_of` can be called both ways, you should also test the type for `a`.

Reply:
Thanks!